internal/infrastructure/storage/db/test/stats_repository_impl_test.go

```diff
@@ -0,0 +1,118 @@
+package db
```

Comment:
Please stick with the code structure of other test files.

Reply:
Done